msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2024-08-18 09:06+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI +ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0\n"
"Language-Code: en\n"
"Language-Name: English\n"
"Preferred-Encodings: utf-8 latin1\n"
"Domain: DOMAIN\n"

#: ../validators/IdValidator.py:26
msgid "Id $id is already in use"
msgstr "Идентификатор $id вече се използва"

#: ../validators/IdValidator.py:17
msgid "Spaces are not allowed in ids"
msgstr "Интервалите не са разрешени във идентификаторите"

#: ../validators/RegexValidator.py:46
msgid "Validation failed($name): $value of type $type, expected 'string'"
msgstr "Проверката е неуспешна($name): $value от тип $type, очаква се 'string'"

#: ../validators/RegexValidator.py:65
msgid "Validation failed($name): '$value' $errmsg"
msgstr "Проверката е неуспешна($name): '$value' $errmsg"

#: ../validators/EmptyValidator.py:45
msgid "Validation failed($name): '$value' is not empty."
msgstr "Проверката е неуспешна($name): '$value' не е празно."

#: ../validators/RangeValidator.py:41
msgid "Validation failed($name): '$value' out of range($min, $max)"
msgstr "Проверката е неуспешна($name): '$value' извън диапазон($min, $max)"

#: ../validators/SupplValidators.py:103
msgid "Validation failed($name): could not convert $value to a date."
msgstr "Проверката е неуспешна($name): '$value' не може да се преобразува в дата."

#: ../validators/RangeValidator.py:33
msgid "Validation failed($name): could not convert '$value' to number"
msgstr "Проверката е неуспешна($name): '$value' не може да се преубразува в число"

#: ../validators/SupplValidators.py:94
msgid "Validation failed($name): value is empty ($value)."
msgstr "Проверката е неуспешна($name): стойността е празна ($value)."

#: ../validators/SupplValidators.py:71
msgid "Validation failed($name: Uploaded data is too large: ${size}MB (max ${max}MB)"
msgstr "Проверката е неуспешна($name): Качения файл е прекалено голям: ${size}MB (максимум ${max}MB)"

#: ../validators/BaseValidators.py:51
msgid "contains unprintable characters"
msgstr "съдържа символи, които не могат да бъдат отпечатани"

#: ../validators/BaseValidators.py:37
msgid "is not a decimal number."
msgstr "не е десетично число."

#: ../validators/BaseValidators.py:95
msgid "is not a valid email address."
msgstr "не е валиден email адрес."

#: ../validators/BaseValidators.py:74
msgid "is not a valid international phone number."
msgstr "не е валиден международен телефонен номер"

#: ../validators/BaseValidators.py:88
msgid "is not a valid url."
msgstr "не е валиден url."

#: ../validators/BaseValidators.py:66
msgid "is not a valid us phone number."
msgstr "не е валиден американски телефонен номер."

#: ../validators/BaseValidators.py:81
msgid "is not a valid zip code."
msgstr "не е валиден зип код."

#: ../validators/BaseValidators.py:58
msgid "is not a well formed SSN."
msgstr ""

#: ../validators/BaseValidators.py:44
msgid "is not an integer."
msgstr "не е цяло число."

#: ../validators/BaseValidators.py:109
msgid "this name is not a valid identifier"
msgstr "това не е валиден идентификатор"

#: ../validators/ExpressionValidator.py:94
msgid "validation failed, expr was:$expr"
msgstr ""
